e2e-tests/cypress/e2e/journeys/insurance/your-business/nature-of-business/nature-of-business-page.spec.js

```diff
@@ -25,13 +30,15 @@ context('Insurance - Your business - Nature of your business page - As an Export
 
     cy.submitInsuranceEligibilityAndStartApplication();
 
+    task.link().click();
+
+    cy.completeCompanyDetails();
```

Comment:
Do you think we should have `completeAndSubmit` prefix? Done this in other pages but happy to not do this